roles/taca/tasks/main.yml

```diff
@@ -46,3 +57,10 @@
 - name: Deploy sthlm_fastq_taca_delivery.yml config
   template: src="sthlm_fastq_taca_delivery.yml.j2" dest="{{ ngi_pipeline_conf }}/TACA/sthlm_taca_fastq_delivery.yml"
   tags: taca
+
+- set_fact:
+    proj_path: "/pica/sw/links/proj/"
+
+- name: Deploy sthlm_fake_delivery.yml config
```

Comment:
I would have assumed that the "fake delivery" would be the within-Irma delivery with `proj_path: /proj` and the regular config pointing to `/pica/...` but it seems to be the other way around, is that the intention?